### ISSUE TITLE
Error loading, works after app restart

### ISSUE THREAD

**Issue by JeFawk** [Scirra/Construct-3-bugs#858]
## Problem description

First time starting the C3 desktop app  (saved from google chrome) after starting up my computer I get this error message. Once I close it and restart it, it works.

I've noticed this happening since r58.

## Attach a .c3p

There is none, I just open the C3 chrome desktop app.

![image](https://user-images.githubusercontent.com/7268697/31018693-a638a79c-a535-11e7-88e0-2de71ab8f8da.png)

## Affected browsers

* Chrome: yes

## System details

*If you see a crash report dialog, please copy and paste it to where it says "PASTE HERE" below. Otherwise please go to **Menu** -> **About** -> **Platform information** and paste that information there instead.*

<details><summary>View details</summary>

Platform information
Browser: Chrome
Browser version: 61.0.3163.100
Browser engine: Blink
Browser architecture: 64-bit
Context: webapp
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 0.8999999761581421
Logical CPU cores: 4
Approx. device memory: (unavailable)
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36
C3 release: r59
Language setting: en-US

Local storage
Storage quota (approx): 6.7 gb
Storage usage (approx): 559 mb (8.1%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Vendor: Google Inc.
Renderer: ANGLE (NVIDIA GeForce GTX 770 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:

EXT_color_buffer_float
EXT_disjoint_timer_query_webgl2
EXT_texture_filter_anisotropic
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by AshleyScirra** [comment]
Never seen this myself and there are no steps to reproduce so not sure there's anything we can do here.

**Comment by JeFawk** [comment]
I understand, I was thinking maybe you have an idea what could cause it. I'd go ahead and reinstall chrome but it's not really worth it right now :)

**Comment by JeFawk** [comment]
Had this crash now instead of the red error text, maybe it helps

Error report information
Type: unhandled rejection
Reason: Error: Failed to fetch @ TypeError: Failed to fetch at fetch.then (https://editor.construct.net/r59/main.js:2:192940) at 
Stack: TypeError: Failed to fetch at fetch.then (https://editor.construct.net/r59/main.js:2:192940) at 
Construct 3 version: r59
URL: https://editor.construct.net/index.html
Date: Wed Oct 04 2017 13:57:05 GMT+0300 (GTB Daylight Time)
Uptime: 3.7 s

Unable to collect further diagnostic information because a promise rejection occurred while trying to collect it.

**Comment by AshleyScirra** [comment]
So it seems one of the scripts is failing to load, which would explain the earlier error too. When it happens can you open the browser error console and see what it says there?

**Comment by JeFawk** [comment]
Got it to throw the red error message again. The font errors are common, the others ones are new.

![image](https://user-images.githubusercontent.com/7268697/31227017-fb351bf4-a9e0-11e7-8cff-e469360e5882.png)

**Comment by AshleyScirra** [comment]
Most of those errors don't look related to anything C3 does. Do you have any browser addons? Does it still happen if you disable them?

**Comment by JeFawk** [comment]
Seeing as this happens only once a day, I'll try that tomorrow, though kinda weird how it started happening all of a sudden.

**Comment by AshleyScirra** [comment]
Closing as probable duplicate of #911, which is hopefully fixed in the next release.
